process/transaction/baseProcess.go

```diff
@@ -341,7 +342,20 @@ func (txProc *baseTxProcessor) GetSenderAndReceiverAccounts(tx *transaction.Tran
 		return nil, nil, process.ErrNilTransaction
 	}
 
-	return txProc.getAccounts(tx.SndAddr, tx.RcvAddr)
+	accSnd, accRcv, err := txProc.getAccounts(tx.SndAddr, tx.RcvAddr)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	if common.IsRelayedTxV3(tx) {
```

Comment:
this is not good.

The sender is the sender. the Payer for the GAS is the relayer.

Reply:
make the implementation needed in the isTransactionEligibleForExecution function.